charts/mimir-distributed/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 2.0.8
+version: 2.0.9
```

Comment:
Please update the CHANGELOG as well

Reply:
done @ 259d65c33bcd922c12157cc530c7eb5169afc492